docs/pages/build/framework/batch.mdx

```diff
@@ -0,0 +1,32 @@
+# Batch
+
+Batches are structured collections of blocks within a blockchain. It includes multiple blocks covering specific height ranges along with transaction hash and DA data. Confirmation statuses from ``C3A`` and beyond are fetched from batches.
+Also, batches gets settled on L1 every 6 hours.
```

Comment:
this seems wrong, they instead of `structured collections` we should just say `ordered set` maybe. I would rewrite this line to write more on "What Batches solve" in subtle way rather than what they have.
Also, we should write `by default Vulcan is configured to settle Batches every 6 hours or when max blob size is reached, whichever happens first.`
cc: @0xRampey 

you can give more insights on this paragraph

Reply:
Batch is a sequence of one or more linked blocks with some metadata.
Main motivation for using batch is
- Cost savings:  Cost of settlement can be amortized across `n` blocks by batching them together and submitting a single transaction to L1, instead of n separate txs
- Composability: Vulcan can batch blocks from different MRUs to potentially support atomic execution b/w MRUs and provide composability